src/library/LibraryWindow.vala

```diff
@@ -513,8 +513,8 @@ public class LibraryWindow : AppWindow {
 
     protected override void on_quit () {
         window_settings.set_boolean ("library-maximize", is_maximized);
-        window_settings.set_int ("library-width", dimensions.width);
-        window_settings.set_int ("library-height", dimensions.height);
+        window_settings.set ("library-position", "(ii)", out window_x, out window_y);
+        window_settings.set ("library-size", "(ii)", out width, out height);
```

Comment:
You should not have the "out" keyword when setting - you are not retrieving values out of the settings - you are putting them in.